data/adapters/SEM_prediction_adapter.py

```diff
@@ -0,0 +1,116 @@
+                           'binding_created', 'binding_increased']  # ignore negative cases
+
+    OUTPUT_PATH = './parsed-data'
+
+    def __init__(self, filepath, label='sem_predicted_asb', dry_run=True):
+        if label not in SEMPred.ALLOWED_LABELS:
+            raise ValueError('Ivalid label. Allowed values: ' +
+                             ','.join(SEMPred.ALLOWED_LABELS))
+
+        self.filepath = filepath
+        self.label = label
+        self.dataset = label
+        self.type = 'edge'
+        self.dry_run = dry_run
+        self.output_filepath = '{}/{}.json'.format(
+            self.OUTPUT_PATH,
+            self.dataset
+        )
+        super().__init__()
+
+    def load_tf_id_mapping(self):
+        self.tf_id_mapping = {}
+        with open(SEMPred.TF_PROTEIN_MAPPING_PATH, 'r') as map_file:
+            map_csv = csv.reader(map_file)
+            for row in map_csv:
+                if row[2]:  # this is a complex
+                    self.tf_id_mapping[row[0]] = 'complexes/' + row[2]
+                else:
+                    self.tf_id_mapping[row[0]] = 'proteins/' + row[3]
+
+    def process_file(self):
+        parsed_data_file = open(self.output_filepath, 'w')
+        self.load_tf_id_mapping()
+        for filename in os.listdir(self.filepath):
+            if filename.endswith('.tsv'):
+                tf_name = filename.split('.')[0].split('_')[0]
+                tf_id = self.tf_id_mapping.get(tf_name)  # protein or complex
+                if tf_id is None:
+                    continue
+                with open(self.filepath + '/' + filename, 'r') as sem_file:
+                    sem_csv = csv.reader(sem_file, delimiter='\t')
+                    for row in sem_csv:
+                        if row[0].startswith('#'):
+                            continue
+
+                        if row[-1] in SEMPred.BINDING_EFFECT_LIST:
+                            chr = row[0]
+                            pos = row[2]  # 1-based coordinate
+                            ref = row[4]
+                            alt = row[5]
+
+                            variant_id = build_variant_id(
+                                chr, pos, ref, alt, 'GRCh38')
+
+                            _key = variant_id + '_' + \
+                                tf_id.split('/')[-1] + '_' + SEMPred.SOURCE
+                            _from = 'variants/' + variant_id
+                            _to = tf_id
+
+                            _props = {
+                                '_key': _key,
+                                '_from': _from,
+                                '_to': _to,
+                                'label': 'predicted allele specific binding',
+                                'motif': 'motifs/' + tf_name + '_' + SEMPred.SOURCE,
+                                'kmer_chr': row[6].split(':')[0],
+                                'kmer_start': int(row[6].split(':')[-1].split('-')[0]),
```

Comment:
I think `kmer_start` need to be updated to `kmer_start:long`. Same with `kmer_end`